targets/targets.json

```diff
@@ -584,7 +584,9 @@
         "features": ["LWIP", "STORAGE"],
         "release_versions": ["2", "5"],
         "device_name": "MK64FN1M0xxx12",
-        "bootloader_supported": true
+        "bootloader_supported": true,
+        "vector_table_size": 1024,
+        "flash_erase_size": 4096
```

Comment:
arm-pack-manager is a library that extracts information in a more readable format from cmsis packs. The flash IAP source file generating script uses arm-pack-manager.

So to directly answer your question: No. But that would not tell the whole story.